ClientApp/src/app/app.module.ts

```diff
@@ -1,8 +1,10 @@
 import { BrowserModule } from '@angular/platform-browser';
-import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { NgModule } from '@angular/core';
+
+import { AppRoutingModule } from './app-routing.module';
```

Comment:
This should be moved lower to separate framework and libraries imports from application code imports.